README.md

```diff
@@ -91,12 +96,19 @@ matches the constraints from the manifest. If the dependency is missing from
 
 ### Adding a dependency
 
-1. `import` the package in your `*.go` source code file(s).
-1. Run the following command to update your `Gopkg.lock` and populate `vendor/` with the new dependency.
+Adding a project as a dependency has three essential steps:
 
-    ```sh
-    $ dep ensure
-    ```
+1. `import` a package from the project in one of your `*.go` source files
+2. Add a version constraint on the project to `Gopkg.toml` (Optional, but recommended)
+3. Run `dep ensure`
+
+`dep ensure -add` provides some CLI sugar to ease this process:
```

Comment:
the only core behavior missing from ensure's implementation is hoisting a constraint up out of the lock. maybe i should just blow some time on getting that last bit of ensure done tonight 🤔 

even then, though, that's the problem - `dep ensure -add` doesn't actually replace step 1. it's just a stopgap for it.